synchrad/calc.py

```diff
@@ -305,29 +306,29 @@ def _process_track(self, particleTrack, comp, nSnaps, it_range):
         args_track = [coord.data for coord in (x, y, z, ux, uy, uz)]
         args_track += [wp, it_start, np.uint32(it_range[-1]), np.uint32(x.size)]
 
-        if self.Args['mode'] is 'far':
+        if self.Args['mode'] == 'far':
```

Comment:
For string (and other immutable types such as ints) comparison, `==` is the right choice, `is` is used to see if two objects point to the same class instance.

Reply:
yeah, I've also noticed this new warning and had even prepared a fix but never got time to push it. Thanx for doing it =)